libexec/std/internal/output.bash

```diff
@@ -0,0 +1,251 @@
+#
+# Globals:
+#   none
+# Arguments:
+#   $1 - column width
+#   $even - key
+#   $odd - value of the previous key
+# Returns:
+#   none
+# Outputs:
+#   STDOUT - key-value pairs in two-column format
+_print_kv_single () {
+  local -ir col_width="$1"; shift
+  while (( $# != 0 )); do
+    printf '%-*s : %s\n' "$col_width" "$1" "$2"
+    shift 2
+  done
+}
+
+# Print key-value pairs in multi-line format. First, the key and the number of
+# lines in the value is printed. Next, the value on a separate line. Every
+# odd-numbered parameter is a key and the following parameters is its value.
+#
+# Globals:
+#   none
+# Arguments:
+#   $odd - key
+#   $even - value of the previous key
+# Returns:
+#   none
+# Outputs:
+#   STDOUT - key-value pairs in multi-line format
+_print_kv_multi () {
+  while (( $# != 0 )); do
+    printf '%s (%d lines):\n' "$1" "$( _count_lines "$2" )"
+    printf '%s\n' "$2"
+    shift 2
+  done
+}
+
+# Print all key-value pairs in either two-column or multi-line format. If all
+# values are one line long, all pairs are printed in two-column format using
+# `_print_kv_single()'. Otherwise, they are printed in multi-line format using
+# `_print_kv_multi()' after each line of all values being prefixed with two
+# spaces.
+#
+# Globals:
+#   none
+# Arguments:
+#   $1 - column width for two-column format
+#   $even - key
+#   $odd - value of the previous key
+# Returns:
+#   none
+# Outputs:
+#   STDOUT - key-value pairs in two-column format, if values are single-line
+#            key-value pairs in multi-line format, otherwise
+_print_kv_single_or_multi () {
+  local -ir width="$1"; shift
+  local -a pairs=( "$@" )
+
+  local -a values=()
+  local -i i
+  for (( i=1; i < ${#pairs[@]}; i+=2 )); do
+    values+=( "${pairs[$i]}" )
+  done
+
+  if _is_single_line "${values[@]}"; then
+    _print_kv_single "$width" "${pairs[@]}"
+  else
+    local -i i
+    for (( i=1; i < ${#pairs[@]}; i+=2 )); do
+      pairs[$i]="$( _prefix < <(printf '%s' "${pairs[$i]}") )"
+    done
+    _print_kv_multi "${pairs[@]}"
+  fi
+}
+
+# Prefix each line of the input with an arbitrary string.
+#
+# Globals:
+#   none
+# Arguments:
+#   $1 - [opt = '  '] prefix string
+# Returns:
+#   none
+# Inputs:
+#   STDIN - lines to prefix
+# Outputs:
+#   STDOUT - prefixed lines
+_prefix () {
+  local -r prefix="${1:-  }"
+  local line
+  while IFS='' read -r line || [[ -n $line ]]; do
+    printf '%s%s\n' "$prefix" "$line"
+  done
+}
+
+# Mark select lines of the input by overwriting their first few characters with
+# an arbitrary string. Usually, the input is indented by spaces first using
+# `_prefix()'.
+#
+# Globals:
+#   none
+# Arguments:
+#   $1 - marking string
+#   $@ - zero-based indices of lines to mark
+# Returns:
+#   none
+# Inputs:
+#   STDIN - lines to work on
+# Outputs:
+#   STDOUT - lines after marking
+_mark () {
+  local -r symbol="$1"; shift
+  # Sort line numbers.
+  set -- $( sort -gu <<< "$( printf '%d\n' "$@" )" )
```

Comment:
I replaced `-g` with `-n` (both work the same in our case).
Thank you for diving through the source and catching this issue!